.eslintrc

```diff
@@ -49,32 +58,60 @@
         "no-unused-expressions": 2,
         "no-with": 2,
         "radix": 2,
+        "vars-on-top": 1,
```

Comment:
> @bentrm Do you know if its ok to have empty lines between the variable definitions?

It's actually checked to be the case before any other statement other than another variable definition or a inline comment.